spec/unit/views/index_as_blog_spec.rb

```diff
@@ -0,0 +1,76 @@
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.